espresso-server/library/build.gradle.kts

```diff
@@ -0,0 +1,94 @@
+plugins {
+    id("com.android.library")
+    kotlin("android")
+}
+
+val appiumCompileSdk: String by project
+val appiumMinSdk: String by project
+val appiumTargetSdk: String by project
+val appiumBuildTools: String by project
+val appiumSourceCompatibility: String by project
+val appiumTargetCompatibility: String by project
+val appiumJvmTarget: String by project
+val appiumKotlin: String by project
+val appiumAndroidxTestVersion: String by project
+val appiumAnnotationVersion: String by project
+val appiumComposeVersion: String by project
+val appiumGsonVersion: String by project
+val appiumEspressoVersion: String by project
+val appiumNanohttpdVersion: String by project
+val appiumRobolectricVersion: String by project
+val appiumJUnitVersion: String by project
+val appiumUiAutomatorVersion: String by project
+
+android {
+    compileSdk = appiumCompileSdk.toInt()
+    buildToolsVersion = appiumBuildTools
+    namespace = "io.appium.espressoserver.lib"
+
+    defaultConfig {
+        minSdk = appiumMinSdk.toInt()
+    }
+
+    compileOptions {
+        sourceCompatibility = JavaVersion.valueOf(appiumSourceCompatibility.uppercase())
+        targetCompatibility = JavaVersion.valueOf(appiumTargetCompatibility.uppercase())
+    }
+
+    packaging {
+        resources.excludes.add("META-INF/**")
+    }
+
+    kotlinOptions {
+        jvmTarget = appiumJvmTarget
+    }
+
+    lint {
+        targetSdk = appiumTargetSdk.toInt()
+    }
+
+    testOptions {
+        targetSdk = appiumTargetSdk.toInt()
+        unitTests.isReturnDefaultValues = true
+    }
+}
+
+dependencies {
+    // additionalAppDependencies placeholder (don't change or delete this line)
```

Comment:
Do you think `api` is better for here than `implementation`?
https://github.com/appium/appium-espresso-driver/blob/b94d75864da3f96fec3b58358996ca0c735753c4/lib/server-builder.js#L151

Reply:
@KazuCocoa I changed the script to add `additionalAppDependencies` as `api` ones.